### ISSUE TITLE
test_checker/test_no_output_checker fail on aarch64 and ppc64le

### ISSUE THREAD

**Issue by StefanBruens** [MiniZinc/libminizinc#600]
```
[   27s] =========================== short test summary info ============================
[   27s] FAILED tests/test_solving.py::TestChecker::test_checker - minizinc.error.Mini...
[   27s] FAILED tests/test_solving.py::TestChecker::test_no_output_checker - minizinc....
[   27s] ======= 2 failed, 48 passed, 1 skipped, 8 deselected, 1 warning in 8.02s =======
```

```
[   27s] ___________________________ TestChecker.test_checker ___________________________
[   27s] 
[   27s] self = <test_solving.TestChecker testMethod=test_checker>
[   27s] 
[   27s]     def test_checker(self):
[   27s]         try:
[   27s]             checker = tempfile.NamedTemporaryFile(
[   27s]                 prefix="checker_test_", suffix=".mzc.mzn", delete=False
[   27s]             )
[   27s]             checker.write(b'output["SIMPLE CHECK"];\n')
[   27s]             checker.close()
[   27s]             self.instance.add_file(checker.name)
[   27s]             assert self.instance.method == Method.SATISFY
[   27s] >           result = self.instance.solve()
[   27s] 
[   27s] tests/test_solving.py:55: 
[   27s] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[   27s] ../../BUILDROOT/python-minizinc-0.7.0-4.1.aarch64/usr/lib/python3.8/site-packages/minizinc/instance.py:186: in solve
[   27s]     return asyncio.run(coroutine)
[   27s] /usr/lib64/python3.8/asyncio/runners.py:44: in run
[   27s]     return loop.run_until_complete(main)
[   27s] /usr/lib64/python3.8/asyncio/base_events.py:616: in run_until_complete
[   27s]     return future.result()
[   27s] ../../BUILDROOT/python-minizinc-0.7.0-4.1.aarch64/usr/lib/python3.8/site-packages/minizinc/instance.py:250: in solve_async
[   27s]     async for result in self.solutions(
[   27s] _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
[   27s] 
[   27s] self = <minizinc.instance.Instance object at 0xffff918e70a0>, timeout = None
[   27s] nr_solutions = None, processes = None, random_seed = None, all_solutions = False
[   27s] intermediate_solutions = False, free_search = False, optimisation_level = None
[   27s] verbose = False, debug_output = None, kwargs = {}, method = <Method.SATISFY: 1>
[   27s] cmd = ['--output-mode', 'json', '--output-time', '--output-objective', '--output-output-item', '--statistics', ...]
[   27s] files = [PosixPath('/tmp/checker_test_3iyhihiu.mzc.mzn'), PosixPath('/tmp/mzn_fragment3q5stucn.mzn')]
[   27s] solver = 'org.gecode.gecode@6.3.0', status = <Status.UNKNOWN: 2>
...
[   27s]             # Raise error if required
[   27s]             stderr = await read_stderr
[   27s]             if code != 0 or status == Status.ERROR:
[   27s] >               raise parse_error(stderr)
[   27s] E               minizinc.error.MiniZincError: UNKNOWN EXCEPTION.
[   27s] 
[   27s] ../../BUILDROOT/python-minizinc-0.7.0-4.1.aarch64/usr/lib/python3.8/site-packages/minizinc/instance.py:465: MiniZincError
```

This is apparently an unhandled error/exception in minizinc itself:
https://github.com/MiniZinc/libminizinc/blob/0b19fb039afff495832ae48f4c206f2653862bb4/minizinc.cpp#L71
```
    } catch (...) {
      if (slv.getFlagVerbose()) {
        std::cerr << std::endl;
      }
      std::cerr << "  UNKNOWN EXCEPTION." << std::endl;
    }
```

Trying to reproduce this with minizinc directly has not provided any insight, the following two calls (which should be equivalent to the wrapped calls from python) succeed:
```
cat > t1.mzn <<EOF
var 1..5: x;
output["SIMPLE CHECK"];
EOF
cat > t2.mzn <<EOF
var 1..5: x;
int: data :: add_to_output = 2;
EOF
minizinc t1.mzn --output-mode json --output-time --output-objective --output-output-item --statistics
minizinc t2.mzn --output-mode json --output-time --output-objective --output-output-item --statistics
```

**Comment by Dekker1** [comment]
Should be fixed in 76e7e9d32e642c099a910c6192af6f3f8d209a82

**Comment by cyderize** [comment]
This should now be fixed in version 2.7.0
